### ISSUE TITLE
SendgridEmailMessage to: field treated as one email address, but it's a tuple

### ISSUE THREAD

**Issue by devinvenable** [resmio/django-sendgrid#8]
Django.core Email to: is defined as:
to: A list or tuple of recipient addresses.

In SendGridEmailMessage send, the tuple is stringified with: 
   'email': str(self.to),

Since we're passing a UUID that is supposed to uniquely identify a single email, shouldn't self.to be parsed, and for each email address in the list, a separate Email record, each with its own unique UUID, be used?

**Comment by JannKleen** [comment]
Good catch! Your solutions sounds like a good choice, although I have to check out how to implement it.
